dowhy/causal_refuters/bootstrap_refuter.py

```diff
@@ -1,184 +1,208 @@
+                                      self._target_estimand.instrumental_variables + \
+                                      self._estimate.params['effect_modifiers']
+
+        if 'logging_level' in kwargs:
+            logging.basicConfig(level=kwargs['logging_level'])
+        else:
+            logging.basicConfig(level=logging.INFO)
+        self.logger = logging.getLogger(__name__)
+        
+        # Sanity check the parameters passed by the user
+        # If the data is invalid, we throw the corresponding error
+        if required_variables is int:
+            if len(self._variables_of_interest) < required_variables:
+                self.logger.error("Too many variables passed.\n The number of  variables is: {}.\n The number of variables passed: {}".format(
+                    len(self._variables_of_interest),
+                    required_variables )
+                )
+                raise ValueError("The number of variables in the arguments is greater than the number of variables")
+
+        elif required_variables is list:
+            for variable in required_variables:
+                # Find out if the user wants to select or deselect the variable
+                if self._invert is None:
+                    if variable[0] == '-':
+                        self._invert = True
+                    else:
+                        self._invert = False
+
+                if self._invert is True:
+                    if variable[0] != '-':
+                        self.logger.error("The first argument is a deselect {}. And the current argument {} is a select".format(required_variables[0], variable))
+                        raise ValueError("It appears that there are some select and deselect variables by the user. Note you can either select or delect variables at a time not both")
+
+                    if variable[1:] not in self._variables_of_interest:
+                        self.logger.error("The variable {} is not in {}".format(variable, self._variables_of_interest))
+                        raise ValueError("The variable selected by the User is not a confounder, Instrument Variable or a Effect Modifier")
+                else:
+                    if variable[0] == '-':
+                        self.logger.error("The first argument is a select {}. And the current argument {} is a deselect".format(required_variables[0], variable))
+                        raise ValueError("It appears that there are some select and deselect variables by the user. Note you can either select or delect variables at a time not both") 
+
+                    if variable not in self._variables_of_interest:
+                        self.logger.error("The variable {} is not in {}".format(variable, self._variables_of_interest))
+                        raise ValueError("The variable selected by the User is not a confounder, Instrument Variable or a Effect Modifier")    
+        
+        elif required_variables is None:
```

Comment:
As discussed, change the default behavior to True. Also include a condition for False (changing no variables)